grafana/mattermost-mobile-performance.json

```diff
@@ -0,0 +1,2081 @@
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "Total",
+          "useBackend": false
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "disableTextWrap": false,
+          "editorMode": "code",
+          "expr": "sum(mattermost_mobileapp_mobile_team_switch_count{platform=\"android\"})",
+          "fullMetaSearch": false,
+          "hide": false,
+          "includeNullMetadata": true,
+          "instant": false,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "Android",
+          "useBackend": false
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "disableTextWrap": false,
+          "editorMode": "code",
+          "expr": "sum(mattermost_mobileapp_mobile_team_switch_count{platform=\"ios\"})",
+          "fullMetaSearch": false,
+          "hide": false,
+          "includeNullMetadata": true,
+          "instant": false,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "iOS",
+          "useBackend": false
+        }
+      ],
+      "title": "# Events Team Switch",
+      "type": "stat"
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 39,
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "description": "rate for measurements",
+        "hide": 2,
+        "name": "rate",
+        "query": "${VAR_RATE}",
+        "skipUrlSync": false,
+        "type": "constant",
+        "current": {
+          "value": "${VAR_RATE}",
+          "text": "${VAR_RATE}",
+          "selected": false
+        },
+        "options": [
+          {
+            "value": "${VAR_RATE}",
+            "text": "${VAR_RATE}",
+            "selected": false
+          }
+        ]
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "0.95",
+          "value": "0.95"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "multi": false,
+        "name": "percentile",
+        "options": [
+          {
+            "selected": false,
+            "text": "0.99",
+            "value": "0.99"
+          },
+          {
+            "selected": true,
+            "text": "0.95",
+            "value": "0.95"
+          },
+          {
+            "selected": false,
+            "text": "0.90",
+            "value": "0.90"
+          },
+          {
+            "selected": false,
+            "text": "0.75",
+            "value": "0.75"
+          }
+        ],
+        "query": "0.99, 0.95, 0.90, 0.75",
+        "queryValue": "",
+        "skipUrlSync": false,
+        "type": "custom"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-24h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "browser",
+  "title": "Mobile Performance Metrics",
+  "uid": "ddp4txrlmyg3kc",
+  "version": 26,
```

Comment:
I don't know if this version number matters or not, but at least keep in mind we should probably keep incrementing it? I guess it's just a build number :P

Reply:
I guess every time we hit save in the dashboard, it increases. 0/5 on what is the plan for this here, though.

Reply:
It's a good practice to keep it increasing to keep track of changes. But not a huge blocker.